src/common/com/intellij/plugins/haxe/lang/lexer/HaxeTokenTypeSets.java

```diff
@@ -91,6 +91,17 @@
     REGULAR_STRING_PART
   );
 
+  TokenSet KEYWORD_CONSTANTS = TokenSet.create(
+    KFALSE,
+    KTRUE,
+    KNULL
+  );
+
+  TokenSet PSEUDO_KEYWORDS = TokenSet.create(
```

Comment:
I guess SOFT_KEYWORDS a bit better naming (keywords that could be used as declaration name)

Reply:
Done.